<ui version="4.0" >
 <class>Audio</class>
 <widget class="QDialog" name="Audio" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>392</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>484</width>
    <height>392</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>484</width>
    <height>392</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Qsalat - audio</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title" >
    <string>Audio option</string>
   </property>
   <widget class="QPushButton" name="prayerButton" >
    <property name="geometry" >
     <rect>
      <x>360</x>
      <y>20</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fajrButton" >
    <property name="geometry" >
     <rect>
      <x>360</x>
      <y>60</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="prayerLineEdit" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>20</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fajrLineEdit" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>60</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="prayerLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>17</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Regular prayer</string>
    </property>
   </widget>
   <widget class="QLabel" name="fajrLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Fajr prayer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="duaButton" >
    <property name="geometry" >
     <rect>
      <x>360</x>
      <y>100</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="duaLlineEdit" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>100</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="duaLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>97</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Dua</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>160</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title" >
    <string>Play options</string>
   </property>
   <widget class="QCheckBox" name="salatCheckBox" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Play athan audio</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="duaCheckBox" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>30</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Play dua audio</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>230</y>
     <width>461</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title" >
    <string>Player</string>
   </property>
   <widget class="QSlider" name="playSlider" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>60</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>60</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>70</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="volumeLabel" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>69</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton" >
   <property name="geometry" >
    <rect>
     <x>390</x>
     <y>350</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>350</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
